app/forms/BankDetailForms.scala

```diff
@@ -30,6 +30,9 @@ import scala.util.Try
 object BankDetailForms {
 
   val TWO = 2
+  val accountNameRegex = "^[a-zA-Z][a-zA-Z '.& \\/]{1,59}$"
+  val swiftCodeRegex = "^[A-Za-z0-9 \\-]{1,11}$"
+  val ibanRegex = "^[A-Za-z0-9 \\-]{1,34}$"
```

Comment:
Please follow best practices to name constants. Would it be worth moving these to constant file?

Reply:
Fixed it